### ISSUE TITLE
Create new tagged release

### ISSUE THREAD

**Issue by fsaad** [probcomp/Gen.jl#507]
The last release is from June 2022 https://github.com/probcomp/Gen.jl/releases/tag/v0.4.5.

Let's create a new tag `v0.4.6` to reflect these changes

```
262f96f0 (HEAD -> master, origin/master) Merge pull request #501 from mirkoklukas/Type_Signature_Fix
cec0486a Merge pull request #497 from probcomp/20230109-ztangent-fix_dist_dsl_grads
4af5d8df Merge pull request #495 from chentoast/dirichlet
b08cae9d Merge pull request #494 from probcomp/mrb/doc_links_in_readme
69430277 Merge pull request #492 from probcomp/2022-12-14-fsaad-doc
6fe74f84 Merge pull request #488 from probcomp/2022-11-11-fsaad-test-ci
b09aba67 Merge pull request #486 from probcomp/2022-11-08-fsaad-doc-clarification
4bf152d1 Merge pull request #484 from probcomp/20221031-fsaad-github-actions
b959223f Merge pull request #480 from arijitnoobstar/master
bcf35043 Merge pull request #481 from probcomp/20221031-fsaad-minor-fixes
e1fb6eb9 Merge pull request #471 from probcomp/mrb/laplace_doc_fix
8ddb6e58 Merge pull request #472 from ztangent/20220615-ztangent-no_load_generated
```

**Comment by fsaad** [comment]
A minor upgade to `v0.5.0` might be more appropriate since the following PR added a new feature:
https://github.com/probcomp/Gen.jl/pull/495
```
4af5d8df Merge pull request #495 from chentoast/dirichlet
```

**Comment by ztangent** [comment]
Yes, a tagged release is a good idea! I can make the PR if everyone else is okay with it. But I don't think we should do a minor upgrade yet. Julia has this convention that when you do a minor upgrade for pre-1.0 versions, it actually breaks compatibility with other packages that currently have their Gen compatibility set to `0.4`. So I would reserve upgrading to 0.5.0 for a intentionally breaking change.

**Comment by ztangent** [comment]
As an additional thought, I think it might be a good idea to fix this particular issue before making a new release, just because it is a regression introduced by a recent change:
https://github.com/probcomp/Gen.jl/issues/477

I still haven't had time to get around to it though. But I can set aside time to do it this weekend.

**Comment by ztangent** [comment]
Finally released v0.4.6!
